src/guest/start.c

```diff
@@ -1141,9 +1151,11 @@ int start_guest(char *name)
 		}
 		g_strfreev(srv_list);
 	}
-
 	cx = snprintf(p, size, "%s", fixed_cmd);
 	p += cx; size -= cx;
+	uid= get_uid();
+	cx = snprintf(p, size,"-device intel-hda -device hda-duplex,audiodev=android_spk -audiodev id=android_spk,timer-period=5000,driver=pa,server=/run/user/%d/pulse/native,in.fixed-settings=off,out.fixed-settings=off", uid);
+	p += cx; size -= cx;
```

Comment:
move this block code to above extra groups (line 1129) since it is not fixed cmd any more.